distributions/openhab/src/main/resources/runtime/services.cfg

```diff
@@ -14,8 +14,9 @@ threadpool:safeCall=10
 threadpool:thingHandler=5
 
 # Start level definitions
-startlevel:20=dsl:items,managed:item,dsl:things,managed:thing,managed:itemchannellink,dsl:persist,managed:metadata
-startlevel:30=persistence:restore
+ startlevel:20=dsl:items,managed:item,dsl:things,managed:thing,managed:itemchannellink,dsl:persist,managed:metadata
+startlevel:30=persistence:restore,automation:scriptEngineFactories
+startlevel:40=dsl:rules,managed:rule,rules:refresh,rules:dslprovider,automation:scriptEngineFactories
```

Comment:
Why does `automation:scriptEngineFactories` appear here again? Imho, every identifier should only occur at exactly one start level definition.